frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -38,6 +38,11 @@ pods:
           hadoop-metrics2:
             template: {{CONFIG_TEMPLATE_PATH}}/journal-hadoop-metrics2.properties
             dest: {{HDFS_VERSION}}/etc/hadoop/hadoop-metrics2.properties
+          {{#TLS_ENABLED}}
```

Comment:
@gabrielhartmann We really need a solution for this type of redundancy.  It occurs frequently.